### PR TITLE
Handle blank contacts in org listings

### LINKED ISSUE #1388: Render org member email if the name is missing
We should try to render the email address _if_ the position and name are blank.

What we render are member names and positions. It is possible to have members with no names and positions, but their email address is listed... so we end up with blank rows.

### PR DESCRIPTION
**Pull request by drewvolz**
This PR fixes issues in:

- **iOS**: the contact info for a Student Org member was completely blank if no first/last name.
- **Android**: the contact info for a Student Org member contained extra whitespace if no first/last name.

Closes #1388 

~ | Before | After |
-- | -- | -- |
iOS | truth-blank | truth-filled
Android | android-truth-blank | android-truth-filled

### CODE REVIEW

**Review by hawkrives — CHANGES_REQUESTED** [review]
- Please extract this helper function into the helper file. If there is no helper file here, please create one.
- When you extract the function, please change it to accept one parameter, as follows: `(c: ContactPersonType)`. That is, remove the object-as-first-parameter dance.

**Comment by drewvolz** [comment]
> Please extract this helper function into the helper file. If there is no helper file here, please create one.

[`clean-org.js`](https://github.com/StoDevX/AAO-React-Native/blob/master/source/views/student-orgs/clean-org.js) exists but I don't think it would belong in there... do you? I think it might warrant another helper file.

**Comment by hawkrives** [comment]
@drewvolz Instead of making another file, I might just rename `clean-org.js` to `util.js` or something, and add the helper function in there.

Since `clean-org.js` has only one function already.

**Comment by drewvolz** [comment]
I think this is ready to go.

File: source/views/student-orgs/detail.android.js [hunk 1]
```diff
@@ -5,11 +5,11 @@ import moment from 'moment'
 import {Card} from '../components/card'
 import * as c from '../components/colors'
 import type {StudentOrgType} from './types'
-import type {ContactPersonType} from './types'
 import type {TopLevelViewPropsType} from '../types'
 import Communications from 'react-native-communications'
 import openUrl from '../components/open-url'
-import cleanOrg from './clean-org'
+import {cleanOrg} from './util'
+import {showNameOrEmail} from './util'
```
Comment (hawkrives) [inline source/views/student-orgs/detail.android.js]: Please import both of these at once.

File: source/views/student-orgs/detail.ios.js [hunk 2]
```diff
@@ -5,10 +5,10 @@ import moment from 'moment'
 import {Cell, Section, TableView} from 'react-native-tableview-simple'
 import * as c from '../components/colors'
 import type {StudentOrgType} from './types'
-import type {ContactPersonType} from './types'
 import type {TopLevelViewPropsType} from '../types'
 import openUrl from '../components/open-url'
-import cleanOrg from './clean-org'
+import {cleanOrg} from './util'
+import {showNameOrEmail} from './util'
```
Comment (hawkrives) [inline source/views/student-orgs/detail.ios.js]: Please import both of these at once.

**Review by hawkrives — CHANGES_REQUESTED** [review]

**Comment by hawkrives** [comment]
One last thing! It looks good!

**Comment by drewvolz** [comment]
Why don't we import from `types` like that?

**Comment by hawkrives** [comment]
@drewvolz you mean here?

```js
import type {StudentOrgType} from './types'
import type {TopLevelViewPropsType} from '../types'
```

Because they're importing from two different files.

If you see anywhere where there are two import statements targeting the same file, please merge them into one :)

**Comment by drewvolz** [comment]
Good catch. I’ve now merged the import statements you mentioned earlier.

**Review by hawkrives — APPROVED** [review]

**Pull request merged**